### PR TITLE
Remove "improved", above/below for accessibility, capitalization, verb, POV  conformance.

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-1166

### CODE REVIEW

**Review by rmloveland — CHANGES_REQUESTED** [review]
98% LGTM, except:

- In a couple places, I think the edits are not strictly style but cross into removing / changing content
- Everywhere this PR says singular "CockroachDB multiregion capabilities" it should use the plural "CockroachDB's multiregion capabilities"

**Comment by rmloveland** [comment]
meta comment: I think when the style team meets we should talk about PR size.  IMO PRs like this that are large and do multiple "things" (headings and some punctuation and page titles and ..) should ideally be broken up.  The review is kinda tedious for human brains to look at a bunch of strictly style edits (esp. that touch multiple points from the style guide) and it's easy to miss stuff.  Just, like, my opinion of course, but I look forward to discussing with y'all when we convene

**Comment by stbof** [comment]
> meta comment: I think when the style team meets we should talk about PR size. IMO PRs like this that are large and do multiple "things" (headings and some punctuation and page titles and ..) should ideally be broken up. The review is kinda tedious for human brains to look at a bunch of strictly style edits (esp. that touch multiple points from the style guide) and it's easy to miss stuff. Just, like, my opinion of course, but I look forward to discussing with y'all when we convene

Agreed. I got carried away.

File: _includes/v21.2/topology-patterns/multiregion-fundamentals.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+- What are your [survival goals](multiregion-overview.html#survival-goals)?  Do you need to survive a [zone failure](multiregion-overview.html#surviving-zone-failures)? A [region failure](multiregion-overview.html#surviving-region-failures)?
+- What are the [table localities](multiregion-overview.html#table-locality) that will provide the performance characteristics you need for each table's data?
+  - Do you need low-latency reads and writes from a single region? Do I need that single region to be configurable at the [row level](multiregion-overview.html#regional-by-row-tables)? Or will [a single optimized region for the entire table](multiregion-overview.html#regional-tables) suffice?
+  - Do you have a "read-mostly" [table of reference data that is rarely updated](multiregion-overview.html#global-tables), but that must be read with low latency from all regions?
 
-For more information about our multi-region capabilities, review the following pages:
+For more information about CockroachDB multi-region capabilities, review the following pages:
```
[the remark this answers is not in the document]
Comment (stbof) [reply _includes/v21.2/topology-patterns/multiregion-fundamentals.md:8]: I think what I wrote is correct and I'm not a fan of making describing every feature of any technology in possessive terms.  It anthropomorphizes the technology. Perhaps the Style team should take this up.

**Comment by stbof** [comment]
> 98% LGTM, except:
> 
> * In a couple places, I think the edits are not strictly style but cross into removing / changing content
> * Everywhere this PR says singular "CockroachDB multiregion capabilities" it should use the plural "CockroachDB's multiregion capabilities"

That's not plural, that's possessive. I don't think possessive is appropriate, but I think the Style team should decide.

File: v21.2/multiregion-overview.md [hunk 2]
```diff
@@ -1,51 +1,42 @@
...
 ---
 
-## Overview
+This page provides an overview of CockroachDB multi-region features.
 
- CockroachDB has improved multi-region capabilities that make it easier to run global applications.
+## Overview
 
-To take advantage of these improved capabilities, you will need to understand the following concepts:
+CockroachDB multi-region capabilities make it easier to run global applications. To take advantage of these capabilities, you should understand the following concepts:
```
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/multiregion-overview.md:11]: I created this: https://cockroachlabs.atlassian.net/browse/DOC-2433

File: v21.2/multiregion-overview.md [hunk 3]
```diff
@@ -1,51 +1,42 @@
...
-- Regions are specified using the `region` key.
-- Zones through the use of the `zone` key.
-
-For example, the command below adds `us-east-1` to the list of cluster regions, and `us-east-1b` to the list of zones:
+You define a cluster region at the node level using the `region` [node startup locality option](cockroach-start.html#locality). For example, the following command adds `us-east-1` to the list of cluster regions:
 
 {% include copy-clipboard.html %}
 ~~~ shell
-cockroach start --locality=region=us-east-1,zone=us-east-1b # ... other required flags go here
+cockroach start --locality=region=us-east-1 # ... other required flags go here
```
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/multiregion-overview.md:39]: I found it confusing that while this section has the heading **Cluster regions** it defines both regions and zones and includes the zone in the example. Would it make sense to change the heading and then restore the zone info?

File: v21.2/multiregion-overview.md [hunk 4]
```diff
@@ -111,27 +96,22 @@ The region level survival goal has the property that the database will remain fu
...
-{{site.data.alerts.end}}
+- [Regional tables](#regional-tables) provide low-latency reads and writes for an entire table from _a single region_.
+- [Regional by row tables](#regional-by-row-tables) provide low-latency reads and writes for one or more rows of a table from _a single region_. Different rows in the table can be optimized for access from _different regions_.
+- [Global tables](#global-tables) are optimized for low-latency reads from _all regions_.
 
 {{site.data.alerts.callout_info}}
-Table locality settings are used for optimizing latency under different read/write patterns. If you are optimizing for read/write access to all of your tables from a single region (the primary region), there is nothing else you need to do once you set your [database's primary region](#database-regions).
+Regardless of table locality, you can access all data in a multi-region database with low latency from any [database region](#database-regions) by using stale [(follower)](follower-reads.html) reads.
```
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/multiregion-overview.md:114]: Agreed. I found the first version hard to parse and my rewrite also failed. I'll try again.

**Comment by rmloveland** [comment]
Stephanie Bodoff ***@***.***> writes:

> @stbof commented on this pull request.
>
> ----------------------------------------------------------------------
>
> In v21.2/multiregion-overview.md:
>
>>  
> -To take advantage of these improved capabilities, you will need to understand the following concepts:
> +CockroachDB multi-region capabilities make it easier to run global applications. To take advantage of these capabilities, you should understand the following concepts:
>
> I created this: https://cockroachlabs.atlassian.net/browse/DOC-2433

Works for me, thanks for making the issue

**Comment by rmloveland** [comment]
Stephanie Bodoff ***@***.***> writes:

> @stbof commented on this pull request.
>
> ----------------------------------------------------------------------
>
> In _includes/v21.2/topology-patterns/multiregion-fundamentals.md:
>
>>  
> -For more information about our multi-region capabilities, review the following pages:
> +For more information about CockroachDB multi-region capabilities, review the following pages:
>
> I think what I wrote is correct and I'm not a fan of making describing
> every feature of any technology in possessive terms. It
> anthropomorphizes the technology. Perhaps the Style team should take
> this up.

Fair enough - it just "sounds" off to me to say "CockroachDB Feature X"
singular but maybe it's not.  Happy to discuss with the team

**Comment by rmloveland** [comment]
Stephanie Bodoff ***@***.***> writes:

> @stbof commented on this pull request.
>
> ----------------------------------------------------------------------
>
> In v21.2/multiregion-overview.md:
>
>>  
>  {% include copy-clipboard.html %}
>  ~~~ shell
> -cockroach start --locality=region=us-east-1,zone=us-east-1b # ... other required flags go here
> +cockroach start --locality=region=us-east-1 # ... other required flags go here
>
> I found it confusing that while this section has the heading Cluster
> regions it defines both regions and zones and includes the zone in the
> example. Would it make sense to change the heading and then restore
> the zone info?

I'm sure this could be improved (like, for sure), but if this is a
style PR I don't think we should be opportunistically changing stuff
that isn't strictly style and that already had technical review.  Adding
a zone as well as a region is def normal and expected here, but I'm sure
we could update this to do a better job in future.  I just don't think
this PR is where to do it.

FWIW we do say "Regions and zones are defined at the node level: ..."
which is something of an intro to this topic

**Comment by rmloveland** [comment]
Stephanie Bodoff ***@***.***> writes:

> @stbof commented on this pull request.
>
> ----------------------------------------------------------------------
>
> In v21.2/multiregion-overview.md:
>
>>  
>  {{site.data.alerts.callout_info}}
> -Table locality settings are used for optimizing latency under different read/write patterns. If you are optimizing for read/write access to all of your tables from a single region (the primary region), there is nothing else you need to do once you set your [database's primary region](#database-regions).
> +Regardless of table locality, you can access all data in a multi-region database with low latency from any [database region](#database-regions) by using stale [(follower)](follower-reads.html) reads.
>
> Agreed. I found the first version hard to parse and my rewrite also
> failed. I'll try again.

Hey that's totally fair re: hard to parse, I bet there's a lot of
opportunity for rewriting stuff like this across our whole doc set.  We
often use too many words, too awkwardly - esp in v1 of a doc

I think my only objection was removing some of the content (IIRC the
first sentence was removed) - would totes understand if you wanted to
kill the uses of "thing/other thing" in here, which I know we're moving
away from

**Comment by stbof** [comment]
> Stephanie Bodoff ***@***.***> writes:
> @stbof commented on this pull request. ---------------------------------------------------------------------- In v21.2/multiregion-overview.md: > {% include copy-clipboard.html %} ~~~ shell -cockroach start --locality=region=us-east-1,zone=us-east-1b # ... other required flags go here +cockroach start --locality=region=us-east-1 # ... other required flags go here I found it confusing that while this section has the heading Cluster regions it defines both regions and zones and includes the zone in the example. Would it make sense to change the heading and then restore the zone info?
> I'm sure this could be improved (like, for sure), but if this is a style PR I don't think we should be opportunistically changing stuff that isn't strictly style and that already had technical review. Adding a zone as well as a region is def normal and expected here, but I'm sure we could update this to do a better job in future. I just don't think this PR is where to do it. FWIW we do say "Regions and zones are defined at the node level: ..." which is something of an intro to this topic

Agreed. Sorry for overstepping.

**Comment by stbof** [comment]
> Stephanie Bodoff ***@***.***> writes:
> @stbof commented on this pull request. ---------------------------------------------------------------------- In v21.2/multiregion-overview.md: > {{site.data.alerts.callout_info}} -Table locality settings are used for optimizing latency under different read/write patterns. If you are optimizing for read/write access to all of your tables from a single region (the primary region), there is nothing else you need to do once you set your [database's primary region](#database-regions). +Regardless of table locality, you can access all data in a multi-region database with low latency from any [database region](#database-regions) by using stale [(follower)](follower-reads.html) reads. Agreed. I found the first version hard to parse and my rewrite also failed. I'll try again.
> Hey that's totally fair re: hard to parse, I bet there's a lot of opportunity for rewriting stuff like this across our whole doc set. We often use too many words, too awkwardly - esp in v1 of a doc I think my only objection was removing some of the content (IIRC the first sentence was removed) - would totes understand if you wanted to kill the uses of "thing/other thing" in here, which I know we're moving away from

Agreed. Will restore as it was.

File: v21.2/data-domiciling.md [hunk 5]
```diff
@@ -6,13 +6,11 @@ toc: true
 
 As you scale your usage of [multi-region clusters](multiregion-overview.html), you may need to keep certain subsets of data in specific localities. Keeping specific data on servers in specific geographic locations is also known as _data domiciling_.
 
-CockroachDB has basic support for data domiciling in multi-region clusters using the process described below. At a high level, this process involves:
+CockroachDB has basic support for data domiciling in multi-region clusters using the process described in this page. At a high level, this process involves:
```
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/data-domiciling.md]: Fixed.

File: v21.2/multiregion-overview.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/multiregion-overview.md]: Fixed.

File: v21.2/multiregion-overview.md [hunk 6]
```diff
@@ -98,11 +85,9 @@ With the zone level survival goal, the database will remain fully available for
 
 You can configure a database to survive zone failures using the [`ALTER DATABASE ... SURVIVE ZONE FAILURE` statement](survive-failure.html).
 
-{{site.data.alerts.callout_info}}
```
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/multiregion-overview.md:101]: Fixed.

File: v21.2/multiregion-overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/multiregion-overview.md]: Fixed.

File: v21.2/partitioning.md [hunk 7]
```diff
@@ -94,17 +94,17 @@ For instance, consider the database of a global online learning portal that has
...
     PRIMARY KEY (country, id));
 ~~~
 
-**Primary Key Considerations**
+##### Primary key considerations
 
-- The order in which the columns are defined in the primary key is important. The partitions and subpartitions need to follow that order. In the example of the online learning portal, if you define the primary key as `(country, expected_graduation_date, id)`, the primary partition is by `country`, and then subpartition is by `expected_graduation_date`. You cannot skip `country` and partition by `expected_graduation_date`.
+The order in which the columns are defined in the primary key is important. The partitions and subpartitions need to follow that order. In the example of the online learning portal, if you define the primary key as `(country, expected_graduation_date, id)`, the primary partition is by `country`, and then subpartition is by `expected_graduation_date`. You cannot skip `country` and partition by `expected_graduation_date`.
 
 #### Partition using secondary index
```
[the remark this answers is not in the document]
Comment (stbof) [reply v21.2/partitioning.md]: Fixed.

File: _includes/v21.2/topology-patterns/multiregion-fundamentals.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (stbof) [reply _includes/v21.2/topology-patterns/multiregion-fundamentals.md:8]: Also, CockroachDB's is a trademark violation (see https://cockroachlabs.atlassian.net/browse/DOC-2433).

[the remark this answers is not in the document]
Comment (rmloveland) [reply _includes/v21.2/topology-patterns/multiregion-fundamentals.md:8]: hey this seems like a good argument, but also not one I'm really qualified to assess.  happy to go with your version for now and let style team figure it out going forward

**Review by rmloveland — APPROVED** [review]
LGTM - thanks for making all these improvements!  Looking forward to internalizing more of these style guidelines going forward
